trajopt/src/kinematic_terms.cpp

```diff
@@ -175,6 +175,7 @@ CartPoseErrCalculator::CartPoseErrCalculator(tesseract_kinematics::JointGroup::C
   , indices_(std::move(indices))
 {
   assert(indices_.size() <= 6);
+  is_target_active_ = manip_->isActiveLinkName(target_frame_);
```

Comment:
It appears that `CartPoseErrCalculator` was previously never actually checking if the target was active. This was why the tests were failing for me. The error calculations for the Jacobian and the regular error were in the opposite direction of each other. Not sure if this was potentially leading to other issues.